hangman/word_generator.py

```diff
@@ -0,0 +1,17 @@
+from requests import get
+import random
+from typing import List
+
+
+class WordGenerator:
+    def __init__(self, url) -> None:
+        self.api: str = url
+        self.words: List[str] = []
+
+    def get_data(self) -> None:
```

Comment:
get_data sounds like it will return data, could we change it to fetch data maybe ?